src/expr/proof_rule.h

```diff
@@ -1102,6 +1102,13 @@ enum class PfRule : uint32_t
   // ---------------------
   // Conclusion: (Q)
   INT_TRUST,
+  //======== Multiplication with zero factor
```

Comment:
This rule is a special case of MACRO_SR_PRED_INTRO.  We should consider just using MACRO_SR_PRED_INTRO for uniformity here.